.github/workflows/emscripten.yml

```diff
@@ -0,0 +1,68 @@
+# GitHub Actions workflow to run tests on Linux.
+name: "Emscripten"
+
+on:
+  push: {}
+  pull_request: {}
+  schedule:
+    - cron: "0 0 * * *" # At 00:00 daily.
+
+concurrency:
+  group: "${{ github.workflow }}-${{ github.ref }}"
+  cancel-in-progress: true
+
+jobs:
+  test:
+    name: "Emscripten"
+    runs-on: "ubuntu-latest"
+    if: ${{ github.repository_owner == 'libressl' || github.event_name != 'schedule' }}
+    permissions:
+      contents: read
+    steps:
+      - name: "Checkout repository"
+        uses: actions/checkout@v4
+
+      - name: "Setup emsdk"
+        uses: mymindstorm/setup-emsdk@v14
+        with:
+          version: 3.1.60
+
+      - name: "Prepare repository"
+        run: "./autogen.sh"
+
+      - name: "Configure CMake"
+        run: cmake -Bbuild -DCMAKE_CROSSCOMPILING_EMULATOR=$EMSDK/node/18.20.3_64bit/bin/node -DCMAKE_TOOLCHAIN_FILE=$EMSDK/upstream/emscripten/cmake/Modules/Platform/Emscripten.cmake -DENABLE_ASM=OFF
+
```

Comment:
This seems to run fine locally with the latest emsdk. Is it more typical to build as you've done in the github actions by setting the toolchain parameters manually, or use wrappers like emcmake / emconfigure like below? It doesn't seem to be necessary to manually disable ASM either - we probably shouldn't require a switch if this is the proper default for the platform.

```
$ emcmake cmake .. -GNinja
configure: cmake .. -GNinja -DCMAKE_TOOLCHAIN_FILE=/home/bcook/projects/emsdk/upstream/emscripten/cmake/Modules/Platform/Emscripten.cmake -DCMAKE_CROSSCOMPILING_EMULATOR=/home/bcook/projects/emsdk/node/18.20.3_64bit/bin/node
-- The ASM compiler identification is unknown
-- Found assembler: /home/bcook/projects/emsdk/upstream/emscripten/emcc
-- Warning: Did not find file Compiler/-ASM
-- Setting build type to 'Release' as none was specified.
...
```

Reply:
Good point, using emcmake is the recommended way in Emscripten website, I'll change it to use it instead of feeding it the toolchain file manually